client/src/components/App.jsx

```diff
@@ -64,22 +89,57 @@ const App = class extends React.Component {
   }
 
   getReviews() {
-    axios.get('http://localhost:3000/review/1')
+    axios.get('http://localhost:3000/review/6')
       .then((response) => {
+        const responseArray = response.data.reviews;
+        const { currentPage, reviewsPerPage } = this.state;
+        const indexOfLastReviewInitial = currentPage * reviewsPerPage;
+        const indexOfFirstReviewInitial = indexOfLastReviewInitial - reviewsPerPage;
+        const pageNumbers = [];
+
+        for (let i = 1; i <= Math.ceil(responseArray.length / reviewsPerPage); i += 1) {
+          pageNumbers.push(i);
+        }
+
         this.setState({
+          pageNumbers,
+          indexOfLastReview: indexOfLastReviewInitial,
+          indexOfFirstReview: indexOfFirstReviewInitial,
           oneItem: response.data,
+          currentReviews: responseArray.slice(indexOfFirstReviewInitial, indexOfLastReviewInitial),
         });
       })
       .catch((error) => {
-        // eslint-disable-next-line no-console
         console.log(error);
       });
   }
 
   /*--------------------------------*/
-  handleClick(event) {
+  nextButton() {
+    const { currentPage, pageNumbers } = this.state;
     this.setState({
-      currentPage: Number(event.target.id),
+      currentPage: (currentPage === pageNumbers.length) ? currentPage : currentPage + 1,
+    }, this.updateCurrentReviews); // callback here to avoid putting setState inside render()
```

Comment:
Using callback as the second argument in setState. Great job. I believe I have to do this too in the later future. 👍